metadata/layout.conf

```diff
@@ -0,0 +1,3 @@
+masters = gentoo
+cache-formats = md5-dict
```

Comment:
`cache-formats` specifies the supported cache metadata formats. For a new repo, it makes sense to enable only the current md5-dict format.
`thin-manifests` disables ebuild digest information in manifests. This goes together with Git repos, which already contain their own hashes which make them unnecessary (and allows for merging ebuild changes sanely - otherwise you'd have to recompute the digest any time a merge is done, making version control painful).